### DISCUSSION TITLE
Hyprland-like window animations

### DISCUSSION THREAD

**Discussion by musjj** [qtile/qtile#4438] [Ideas]
I'm not sure if this is out-of-scope, but it'd be cool if qtile can support Hyprland-like window animations, at least for its Wayland compositor.

https://wiki.hyprland.org/Configuring/Animations

https://github.com/qtile/qtile/assets/72612857/f7f0f374-a542-4632-b4fa-bc43799ed5f7

**Comment by m-col** [comment]
Those animations are certainly very cool, but it's unlikely qtile will have anything like that in the short term. There is still much work to do in the wayland backend and fancy animations like this would add some friction to its maintainence. But eventually, if it was simple or implemented elsewhere and plugged into qtile, then that would be cool :)
